pipelines/_ingredients/LSSTCam-imSim/DRP.yaml

```diff
@@ -35,31 +35,24 @@ tasks:
         # Target settings are likely common to all object tables
         from lsst.pipe.tasks.match_tract_catalog_probabilistic import MatchTractCatalogProbabilisticTask
         config.match_tract_catalog.retarget(MatchTractCatalogProbabilisticTask)
-        config.match_tract_catalog.columns_ref_flux = [
-            'flux_u', 'flux_g', 'flux_r',
-            'flux_i', 'flux_z', 'flux_y',
+        bands_match = ("u", "g", "r", "i", "z", "y")
+        # Bands to match aperture fluxes on as a fallback if cModel failed
+        bands_fallback = ("r", "i")
```

Comment:
Why `r` & `i`?

Reply:
I didn't want to include every band because then the aperture fluxes would be weighted as much as the CModel fluxes for matching (there is no way to weight each column differently as of yet). It seems unlikely there are many good detections with only u/g aperture fluxes. I suppose there could be z-band dropouts but this was meant to fix a problem with brighter stars.

Reply:
Got it...a comment in the file would be nice.